### PR TITLE
Pseudo code

### PR DESCRIPTION
**Pull request by Marieerin**
Here is my code! 🐌

### CODE REVIEW

File: scripts/scriptscratch.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+//api key 
+mealApp.key = 'cacba147c38b4f0faff0ce178c6edd1f'
+
+
+// gets user information and sends it to the getMeals function
+mealApp.getUserInfo = (userDiet, intollerances, calories, time) => {
+    // listen for diet to be selected and assign that to userDiet
+    // $('.checkbox-pair').on('click',)
+
+    // listen for intollerances to be selected and assign that to inlosserances
```
Comment (georgezikos) [inline scripts/scriptscratch.js]: I see _intolerances_ spelled incorrectly through out.. Just wondering if that will throw us errors?

File: styles/sass/_scratch.scss [hunk 2]
```diff
@@ -3,103 +3,103 @@
...
-}
-
-.main-header--checkbox {
-    font-family: 'Inter', Arial, 'Helvetica Neue', Helvetica, sans-serif;
-    font-size: 2.4rem;
-    font-weight: 600;
-    letter-spacing: 0.05rem;
-    line-height: 1.4;
-}
+// html {
```
Comment (georgezikos) [inline styles/sass/_scratch.scss]: With all of this commented out, what's the move from here?

Comment (georgezikos) [reply styles/sass/_scratch.scss]: Nevermind, I see further down 😅

File: scripts/scriptscratch.js (hunk 1 above)
Comment (Marieerin) [reply scripts/scriptscratch.js]: will fix

**Review by georgezikos — APPROVED** [review]

**Pull request merged**